### PR TITLE
[TACHYON-572, TACHYON-573, TACHYON-575] style fixes

### PR DESCRIPTION
**Pull request by miasantreble**
https://tachyon.atlassian.net/browse/TACHYON-572
https://tachyon.atlassian.net/browse/TACHYON-573
https://tachyon.atlassian.net/browse/TACHYON-575

### CODE REVIEW

**Comment by calvinjia** [comment]
@miasantreble 
Thanks for the patch! In the future please try to open one PR per JIRA ticket unless absolutely necessary. Also, these tickets are new contributor tasks that are to help new developers familiarize themselves with the pull request process. Please limit yourself to beginner tasks and above after you have done two or more PRs.

File: clients/unshaded/src/main/java/tachyon/client/BlockOutStream.java [hunk 1]
```diff
@@ -110,6 +110,13 @@
     mBuffer = ByteBuffer.allocate(Ints.checkedCast(allocateBytes));
   }
 
+  /**
+   * Write out the buffer to the mapped file, will automatically requtest more space
```
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/BlockOutStream.java]: Typo here in request?

Comment (miasantreble) [inline clients/unshaded/src/main/java/tachyon/client/BlockOutStream.java]: fixed,thanks

**Comment by miasantreble** [comment]
@calvinjia sounds good, I was searching for unassigned open tickets and didn't pay attention to the labels. I will be looking for beginner & intermediate tickets in the future, thanks for the reminder

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM.  Please close these issues once your PRs are merged.

**Comment by haoyuan** [comment]
We can’t automatically merge this pull request.

**Pull request merged**